backend/apps/health/views.py

```
@@ -0,0 +1,263 @@
+            "name": "PostgreSQL",
+            "status": "healthy",
+            "details": {},
+        }
+        try:
+            start = time.time()
+            with connection.cursor() as cursor:
+                cursor.execute("SELECT 1")
+                cursor.fetchone()
+            result["details"]["response_time_ms"] = round((time.time() - start) * 1000, 2)
+            result["details"]["connection_count"] = connection.connection.total_changes() if hasattr(connection.connection, 'total_changes') else "N/A"
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)